main/resource-access.bs

```diff
@@ -70,7 +70,101 @@ A Solid data pod MUST conform to the Web Access Control specification [[!WAC]].
 
 ## Cross-Origin Resource Sharing ## {#cors}
 
-Issue: Write Cross-Origin Resource Sharing section.
-
-Draft:
-A Solid data pod MUST conform to the CORS specification [[!FETCH]].
+### Background and Need ### {#cors-need}
+<em>This section is non-normative.</em>
+
+[=Solid apps=] typically access data from multiple sources.
+However,
+Web browsers by default prevent apps that run on one origin
+from accessing data on other origins.
+This cross-origin protection is a security mechanism
+that ensures malicious websites cannot simply read
+your profile or banking details from other websites.
+However, this reasonable default poses a problem
+even for benevolent Solid apps,
+which might have good reasons to access data from different places.
+For instance,
+a Solid app at `https://app.example/`
+would be prevented from accessing data on
+`https://alice-data-pod.example/` or `https://bob-data-pod.example/`,
+even when Alice and Bob have given the user of the app
+their permission to see some of their data.
+
+For cases where the other origins
+have their own access protection mechanism—
+[like within Solid](#wac)—
+the browser's built-in cross-origin protection
```

Comment:
```
mechanism—
[like within Solid](#wac)—
the
```
->
```
mechanism &mdash;
[like within Solid](#wac) &mdash;
the
```

(If edited to one line, there should be a space both preceding and following the `&mdash;` entity.)

Reply:
m-dashes don't take surrounding spaces (The Chicago Manual of Style).

Reply:
Accessible style guide references on the web have changed since the last time I searched this out.  Apparently, AP recommends the surrounding spaces, while CMOS recommends against.  Lesser-known and private style guides go both ways.  Do we have a definitive statement somewhere that Solid docs will follow CMOS?